### PR TITLE
Add a clear, obvious way to remove hotkeys to improve UX

### PR DESCRIPTION
**Pull request by zbrcz**
The solution is based on bradrisse's comment here:
https://github.com/mui-org/material-ui/issues/2932

Based on that thread, it seems to me that there is currently no better way to add the requested button.

### CODE REVIEW

File: src/renderer/ui/components/settings/HotkeyInput.js [hunk 1]
```diff
@@ -10,6 +10,16 @@ const styles = {
     width: '50%',
     float: 'left',
     padding: '4px 6px',
+    position: 'relative',
+  },
+  clearButton: {
+    'background-color': 'Transparent',
```
Comment (MarshallOfSound) [inline src/renderer/ui/components/settings/HotkeyInput.js]: This can just be `backgroundColor` and the value should be all lower case 👍

**Comment by zbrcz** [comment]
Please note that I have amended the commit. I did it using "git push -f", not sure if that's the right way. :)

The requested screenshot:
hotkeyremovebutton

**Comment by jostrander** [comment]
Looks reasonable. Does it also work with the dark theme?

**Comment by zbrcz** [comment]
No, the icon doesn't change color. Thanks for pointing that out, I'll take a look at it.

**Comment by zbrcz** [comment]
I have changed the `<button>` to a `<FlatButton>` and the arrow now changes color according to the theme. How do I submit the change? I get the feeling that _git push -f_ is not the right way.

**Comment by jostrander** [comment]
`git push -f` is the right way if you are squashing your commits every time, otherwise a regular `git push` would do.

**Comment by jostrander** [comment]
@MarshallOfSound this is good to go, just pulled and reviewed.

**Pull request merged**